core/src/test/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionSpec.scala

```diff
@@ -1258,6 +1259,13 @@ class SubmissionSpec(_system: ActorSystem) extends TestKit(_system)
     }
   }
 
+  "Retrysubmission" should "succeed" in withSubmissionTestWorkspaceService { workspaceService =>
+    val req = workspaceService.retrySubmission(subTestData.wsName, SubmissionRetry(RetryAborted), subTestData.submissionTestAbortTwoGoodWorkflows.submissionId)
+    val report = Await.result(req, Duration.Inf)
+    assertResult(subTestData.submissionTestAbortTwoGoodWorkflows.submissionId, "Retried submission should reference original") { report.originalSubmissionId }
+    assert(report.submissionId != report.originalSubmissionId, "We should generate a new submission id")
```

Comment:
could you add something like
```
report.workflows should have size 2
```
to assert that the logic is correct and it's actually finding two workflows to retry?

Also assuming you make changes to override the submitter and userComment, validate those changes here too